### PR TITLE
Intro/Outro cues and AutoDJ Mix Mode improvements

### PR DESCRIPTION
**Pull request by Be-ing**

### CODE REVIEW

File: source/chapters/djing_with_mixxx.rst [hunk 1]
```diff
@@ -324,8 +330,8 @@ compare it to the length of the intro of the next track.
 
 .. _djing-auto-dj:
 
-Using Auto DJ For Automatic Mixing
-==================================
+Auto DJ
```
Comment (Holzhaus) [inline source/chapters/djing_with_mixxx.rst:333]: We should either use "Auto DJ" or "AutoDJ" consistently. I'd prefer the latter.

**Comment by daschuer** [comment]
No, we are going in circles.
We have decided to use Auto DJ for all visible strings in the GUI.
This should be reflected in the manual.

**Review by daschuer — CHANGES_REQUESTED** [review]
Please revert the last commit and use Auto DJ.

**Comment by Holzhaus** [comment]
ping @Be-ing

**Comment by Be-ing** [comment]
> Please revert the last commit and use Auto DJ.

Okay, I have gone back and done the opposite to change all "AutoDJ" to "Auto DJ".

**Comment by Be-ing** [comment]
merge?

**Review by Holzhaus — APPROVED** [review]

**Pull request merged**